EIPS/draft-factory-standard.md

```diff
@@ -0,0 +1,106 @@
+---
+eip: <to be assigned>
+title: Standard factory contract format
```

Comment:
```suggestion
title: Factory contract format
```
EIPs are standards, so it is redundant to call a standard a standard in the title/description.